examples/interface_examples/procthor.md

```diff
@@ -0,0 +1,120 @@
+# Procthor Interface - Creating a CEO for testing multible robots in mutliple enviromnents
+
+This Notebook aims to provide an overview of the ProcThor Interface, by giving an easy-to-understand example on how to 
+use it. We will be executing a simple pick and place plan with hard coded values and store the NEEMs of these 
+experiments within a local memory database. 
+
+Please be aware that this Notebook (currently) works only when you are connected to the network at the university in 
+Bremen, otherwise we will not be able to connect to the REST service and the webserver itself.
```

Comment:
This sounds very problematic with the ci

Reply:
Yes, maybe we just add an easy way out. If you can not reach the server just skip?

Reply:
sounds good

Reply:
however it would be cool to see the result in the doc which will not happen if it is skipped

Reply:
Currently, you can't see the result in the doc anyway